### PR TITLE
Adobe Commerce Cloud REDIS configuration clarification

### LINKED ISSUE #9205: Needs clarification for Adobe Commerce on Cloud regarding L2 caching
# General issue

https://devdocs.magento.com/guides/v2.4/config-guide/cache/two-level-cache.html

## Description:

Add a differentiation between On-Premise and Adobe Commerce on Cloud

## Possible solutions:

For Adobe Commerce on Cloud projects, please follow https://support.magento.com/hc/en-us/articles/360049292532

## Additional information:

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request

This pull request (PR) fixes #9205 

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.3/config-guide/cache/two-level-cache.html
https://devdocs.magento.com/guides/v2.4/config-guide/cache/two-level-cache.html

### CODE REVIEW

File: src/guides/v2.3/config-guide/cache/two-level-cache.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ To reduce the network bandwidth to Redis, we can store cache data locally on eac
 
 Magento stores the hashed data version in Redis, with the suffix ':version' appended to the regular key. In case of an outdated local cache, the data is transferred to the local machine with a cache adapter.
 
+{:.bs-callout-info}
+For Adobe Commerce Cloud it is recommended to follow the configuration best practice and Extended Redis Cache [implementation guide](https://support.magento.com/hc/en-us/articles/360049292532).
```
Comment (dshevtsov) [inline src/guides/v2.3/config-guide/cache/two-level-cache.md:20]:
```suggestion
For {{ site.data.var.ece }}, consider the [Extended Redis cache implementation](https://support.magento.com/hc/en-us/articles/360049292532) best practice.
```

File: src/guides/v2.4/config-guide/cache/two-level-cache.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ To reduce the network bandwidth to Redis, we can store cache data locally on eac
 
 Magento stores the hashed data version in Redis, with the suffix ':hash' appended to the regular key. In case of an outdated local cache, the data is transferred to the local machine with a cache adapter.
 
+{:.bs-callout-info}
+For Adobe Commerce Cloud it is recommended to follow the configuration best practice and Extended Redis Cache [implementation guide](https://support.magento.com/hc/en-us/articles/360049292532).
```
Comment (dshevtsov) [inline src/guides/v2.4/config-guide/cache/two-level-cache.md:20]:
```suggestion
For {{ site.data.var.ece }}, consider the [Extended Redis cache implementation](https://support.magento.com/hc/en-us/articles/360049292532) best practice.
```

**Comment by BarnyShergold** [comment]
Suggested changes accepted with an additional modification.

**Review by hguthrie — APPROVED** [review]

**Comment by hguthrie** [comment]
Using the word "guide" may be a bit of a stretch here, though. It's best to call it a best practice(s) like it does in the support topic; otherwise one might misinterpret and search for something more extensive. 👀  Guide? where?

**Comment by BarnyShergold** [comment]
@hguthrie - How about swapping "recommendation" for "guide" then?

**Comment by dshevtsov** [comment]
'best practice' is a procedure by definition [see [_best practice_ in Meriam-Webster](https://www.merriam-webster.com/dictionary/best%20practice)]
It does not require additional clarifications in this context.

**Comment by hguthrie** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@hguthrie the Pull Request is successfully imported.

**Pull request merged**